### PR TITLE
chore(tests): fix behavior test regex

### PR DESCRIPTION
**Pull request by miroslavstastny**
Fix regexes uses in behavior test definitions.
Changelog intentionally left blank.

### CODE REVIEW

**Review by sophieH29 — APPROVED** [review]

File: test/specs/behaviors/testDefinitions.ts [hunk 1]
```diff
@@ -56,7 +56,7 @@ definitions.push({
 // Example: Adds attribute 'tabIndex=0' to 'anchor' component's part.
 //          Adds attribute 'data-is-focusable=true' to 'anchor' component's part.
 definitions.push({
-  regexp: /Adds attribute '([a-z A-Z -]+)=([a-z 0-9]+)' to '([a-z]+)' component's part\./g,
+  regexp: /Adds attribute '([\w-]+)=([\w\d]+)' to '([\w\d]+)' component's part\./g,
```
Comment (kuzhelov) [inline test/specs/behaviors/testDefinitions.ts]: is it intentional inclusion of digit in the last regex group? Could you, please, suggest which part? In that case we should make it consistent for other similar places as well (however, not sure that we should include digits here, even given it is technically correct)

Comment (kuzhelov) [reply test/specs/behaviors/testDefinitions.ts]: also, please, note that dash ('-') is not included to this component part group's regex - seems that we should use `([\w-]+)` for it

File: test/specs/behaviors/testDefinitions.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ definitions.push({
 
 // Example:  Adds role 'menuitem' to 'anchor' component's part
 definitions.push({
-  regexp: /Adds role '([a-z]+)' to '([a-z]+)' component's part/g,
+  regexp: /Adds role '(\w+)' to '(\w+)' component's part/g,
```
Comment (kuzhelov) [inline test/specs/behaviors/testDefinitions.ts]: this `component's part`'s group regex turns out to be inconsistent across different places where used - it seems that `([\w-]+)` is the expression we should use for it everywhere

File: test/specs/behaviors/testDefinitions.ts [hunk 3]
```diff
@@ -256,7 +256,7 @@ definitions.push({
 
 // Example: Adds attribute 'aria-hidden=true', if there is no 'alt' property provided.
 definitions.push({
-  regexp: /Adds attribute '([\w\-\w \s*]+)=([a-z]+)', if there is no 'alt' property provided\.+/g,
```
Comment (kuzhelov) [inline test/specs/behaviors/testDefinitions.ts:259]: 👍

File: test/specs/behaviors/testDefinitions.ts [hunk 4]
```diff
@@ -277,7 +277,7 @@ definitions.push({
 
 // Example:  Adds attribute 'aria-expanded=true' based on the property 'open' if the component has 'hasSubtree' property.
 definitions.push({
-  regexp: /Adds attribute '([\w\-\w \s*]+)=([a-z]+)' based on the property '([a-z]+)' if the component has '([a-zA-Z]+)' property./g,
+  regexp: /Adds attribute '([\w-]+)=(\w+)' based on the property '(\w+)' if the component has '(\w+)' property\./g,
```
Comment (kuzhelov) [inline test/specs/behaviors/testDefinitions.ts:280]: 👍

**Review by kuzhelov — APPROVED** [review]
just couple comments

File: test/specs/behaviors/testDefinitions.ts (hunk 2 above)
Comment (miroslavstastny) [reply test/specs/behaviors/testDefinitions.ts]: component part changed to `([\w-]+)` everywhere

**Pull request merged**